app/routes/screens/race/Race.js

```diff
@@ -68,32 +72,40 @@ class RacePredictor extends Component {
         })
       )
     }).isRequired,
+    clubMembers: PropTypes.arrayOf(
+      PropTypes.shape({
+        firstname: PropTypes.string,
+        lastname: PropTypes.string,
+        profile: PropTypes.string,
+        country: PropTypes.string
+      })
+    ).isRequired,
     geoloactionActionsCreators: PropTypes.objectOf(PropTypes.func).isRequired
   };
 
   static intToColor(id = 0) {
     // eslint-disable-next-line
     const c = (id & 0x00ffffff).toString(16).toUpperCase();
-    return "00000".substring(0, 6 - c.length) + c;
+    return `#${"00000".substring(0, 6 - c.length)}${c}`;
```

Comment:
i dont know how you pick your color but there is that in RN
https://github.com/react-community/normalize-css-color

u can use hue instead of RGB may be